packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -1210,6 +1240,12 @@ class SearchBar extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.autofocus}
   final bool autoFocus;
 
+  /// The type of action button to use for the keyboard.
```

Comment:
Maybe we can create a {@template ...} ... {@endtemplate} for `TextField.textInputAction` and use {@macro ...} here:)
Flutter style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-templates-and-macros

Reply:
Sure...I tried using one template from TextField but it didn't work

Reply:
Done 👍✅